src/tests/e2etest/E2ETest.cs

```diff
@@ -151,6 +158,35 @@ private T Deserialize<T>(TwinCollection twinCollection)
             }
         }
 
+        /// <summary>
+        /// Validates the local reported object and fails if not in sync with the given reported object.
+        /// </summary>
+        /// <param name="reported">The reported twin to check the local twin against</param>
+        /// <param name="componentName">The name of the component</param>
+        /// <param name="objectName">The object name (Optional)</param>
+        protected void ValidateLocalReported(Object reported, String componentName, String objectName = "")

```

Comment:
Local management is by default disabled. Is the E2E test enabling that and where?

Reply:
Ok, found where in the yml